controllers/movies.js

```diff
@@ -0,0 +1,27 @@
+const movies = require('../movies')
+
+const findTitle = (request, response) => {
+  const foundTitle = movies.find((movie) => movie.title.toLowerCase().includes(request.params.title.toLowerCase()))
+
+  return response.send(foundTitle)
+}
+
+const findDirector = (request, response) => {
+  const foundMovie = movies.filter((movie) => {
+    return movie.directors.filter((dir) => dir.toLowerCase().includes(request.params.director.toLowerCase()))
```

Comment:
You don't need to filter a second time, instead, explore ways of utilizing the find() method for your movies.directors array somewhere in this function, that may help you out.